keras/backend/tensorflow/nn.py

```diff
@@ -787,3 +787,55 @@ def batch_normalization(
         scale=scale,
         variance_epsilon=epsilon,
     )
+
+
+def ctc_batch_cost(
```

Comment:
Let's rename to `ctc_loss` to stay in line with `tf.nn` and `torch.nn`.

Reply:
Done, thanks!